templates/extra/resources.html

```diff
@@ -0,0 +1,293 @@
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#success-6>Once project is live</a></p></li>
+                                </ul>
+                            </li>
+                        </ul>
+                        <!-- Next Steps -->
+                        <ul class="vertical menu content-box" data-accordion-menu>
+                            <li>
+                                <a class="content-header" href="overview-nested">Next Steps</a>
+                                <ul class="menu vertical nested">
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-1>Engaging Investors</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-2>Raising More</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-3>Another Campaign</a></p></li>
+                                    <li><p class="content-sub"><a href={% url 'resources' %}#next-4>Helping others</a></p></li>
+                                </ul>
+                            </li>
+                        </ul>
+                    </ul>
+                </div>
+            </div>
+            <div class="columns small-9 float-right" id="example1" data-something>
+                <div id="overview">
+                    <h3>overview</h3>
+                    <div class="break">
+                        <hr>
+                    </div>
+                    <div id="overview-1">
+                        <h4>What is PerDiem?</h4>
+                        <p>
+                        A platform where you can raise money for your music projects by having fans and people invest in your music. 
+                        </p>
+                    </div>
+                    <div id="overview-2">
+                        <h4>Why should I let my fans invest in my music?</h4>
+                        <p>
+                        The same reason that most businesses allow people to invest in them. When you give a percentage to investors, this creates an incentive for them to fund your project. This also gives them an incentive to make your project successful. The more money you earn, the more they can earn. It's a win-win for everyone. 
+                        </p>
+                    </div>
+                    <div id="overview-3">
+                        <h4>Why Should I trust perdiem?</h4>
+                        <p>
+                        PerDiem is built on trust and integrity. We rely on our relationship with the creative community to operate. This means we must always act in the best interest of artists. 
+                        </p>
+                    </div>
+                    <div id="overview-4">
+                        <h4>How does it all work? </h4>
```

Comment:
nit: Remove space after question mark before `</h4>`. ie. `<h4>How does it all work?</h4>`